### PR TITLE
tune single-node-performance expected_tps

### PR DESCRIPTION
**Pull request by msmouse**
calibration seems off, failing workflows

## Type of Change
- [x] Tests

## Which Components or Systems Does This Change Impact?
- [x] Other (specify) -- performance test

## How Has This Been Tested?

single-node-performance test

## Checklist
- [x] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: testsuite/single_node_performance.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=13100, key=RunGroupKey("coin-transfer"), included_in=LAND_BLOCKING_AND_C | Flow.REPRESENTATIVE),
     RunGroupConfig(expected_tps=40100, key=RunGroupKey("coin-transfer", executor_type="native"), included_in=LAND_BLOCKING_AND_C),
     RunGroupConfig(expected_tps=11700, key=RunGroupKey("account-generation"), included_in=LAND_BLOCKING_AND_C | Flow.REPRESENTATIVE),
-    RunGroupConfig(expected_tps=35000, key=RunGroupKey("account-generation", executor_type="native"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=30600, key=RunGroupKey("account-generation", executor_type="native"), included_in=Flow.CONTINUOUS),
```
Comment (msmouse) [inline testsuite/single_node_performance.py:94]: I bumped this yesterday according to my tests, but it dropped to the original level somehow. reverting.

File: testsuite/single_node_performance.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=1035, key=RunGroupKey("liquidity-pool-swap-stable"), included_in=Flow.CONTINUOUS),
     RunGroupConfig(expected_tps=4570, key=RunGroupKey("liquidity-pool-swap-stable", module_working_set_size=20), included_in=Flow.CONTINUOUS),
 
-    RunGroupConfig(expected_tps=27000, key=RunGroupKey("deserialize-u256"), included_in=Flow.CONTINUOUS),
+    RunGroupConfig(expected_tps=19250, key=RunGroupKey("deserialize-u256"), included_in=Flow.CONTINUOUS),
```
Comment (msmouse) [inline testsuite/single_node_performance.py:159]: @vusirikala you added this yesterday but looks like it consistently fail to meet your number..

Comment (vusirikala) [reply testsuite/single_node_performance.py:159]: I put in the number based on my local execution. If the forge number is close to 19K, we can use that instead. Totally fine.

**Review by zekun000 — APPROVED** [review]

File: testsuite/single_node_performance.py (hunk 2 above)
Comment (igor-aptos) [reply testsuite/single_node_performance.py:159]: for context - CONTINUOUS are not run on PRs, but on main. so having it off is not as urgent. 

but when adding a new workload, that hasn't been calibrated, you should use `waived=true`, and that will run them - but not have test fail - so you can remove it after recalibration. @vusirikala

**Pull request merged**